_config.yml

```diff
@@ -192,7 +192,12 @@ links_layout: block
 # Sidebar Avatar
 # in theme directory(source/images): /images/avatar.gif
 # in site  directory(source/uploads): /uploads/avatar.gif
-#avatar: /images/avatar.gif
+avatar:
+  rul: 
```

Comment:
Mb `url`?

Reply:
omg... Used to remaining useful life...

Reply:
Don't understand here. It is just mistake, right?

Reply:
yes, it is a mistake.